k8s/live/azure-dev/common-application.yml

```diff
@@ -1,8 +1,9 @@
 spring:
-  data:
-    elasticsearch:
-      username: ${ELASTICSEARCH_USER:#{null}}
-      password: ${ELASTICSEARCH_PASSWORD:#{null}}
+  elasticsearch:
+    uris:
+      - http://elasticsearch:9200
```

Comment:
Do we need it if it's the same as ws-commons?
(same below)

Reply:
fixed